src/Compilers/CSharp/Portable/Compilation/SyntaxTreeSemanticModel.cs

```diff
@@ -839,6 +843,11 @@ internal override MemberSemanticModel GetMemberModel(SyntaxNode node)
             return null;
         }
 
+        /// <summary>
+        /// Internal for test purposes only
```

Comment:
>test purposes only [](start = 25, length = 18)

Is this comment correct? This appears to be used in GetOrAddModelForAttribute... can all of the test-only members please add something like "TestOnly" to their names to help readers see where they are used (and not used)?

Reply:
Oh, I see the *property* is for test purposes only (the field is used elsewhere).

---
In reply to: [170039399](https://github.com/dotnet/roslyn/pull/24918#discussion_r170039399) [](ancestors = 170039399)

Reply:
> Oh, I see the property is for test purposes only (the field is used elsewhere). Still, it would be useful to add TestOnly to such members.

Renamed

---
In reply to: [170039556](https://github.com/dotnet/roslyn/pull/24918#discussion_r170039556) [](ancestors = 170039556,170039399)